src/py21cmfast/inputs.py

```diff
@@ -1054,6 +1055,13 @@ class AstroParams(StructWithDefaults):
     UPPER_STELLAR_TURNOVER_INDEX:
         The power-law index associated with the optional upper mass power-law of the stellar-halo mass relation
         (see FlagOptions.USE_UPPER_STELLAR_TURNOVER)
+    SIGMA_LX: float, optional
+        Lognormal scatter (dex) of the Xray luminosity relation.
```

Comment:
Xray luminosity with what? SFR?

Reply:
The mean is a function of SFR, stellar mass and redshift, but the lognormal scatter is constant. I've added this to the description.